### ISSUE TITLE
Clarification: can pub.Database.GetInbox and pub.Database.GetOutbox return an empty collection?

### ISSUE THREAD

**Issue by BenLubar** [go-fed/activity#141]
Is there any downside to returning an empty collection from GetInbox and GetOutbox without even checking the database? The documentation implies that these functions are only used as a source for a collection object to pass to SetInbox and SetOutbox, and that the modified content will always be additive when used with these two methods.

**Comment by cjslep** [comment]
You're correct, the `GetInbox` and `GetOutbox` on the database is for its own `SetInbox` and `SetOutbox`. `go-fed` will prepend any IRIs to any existing `orderedItems` property. If you return an `OrderedCollection` with no `orderedItems` property, it should create a new one.

Note that the `GetInbox` on `FederatingProtocol` and `GetOutbox` on the `CommonBehavior` are meant for handling the HTTP requests from peers and sending a `OrderedCollection` or `OrderedCollectionPage` in the response.

I'm open to suggestions on improvements (API, documentation, etc).
